src/feature/hs/hs_client.c

```diff
@@ -1445,6 +1445,79 @@ client_dir_fetch_unexpected(dir_connection_t *dir_conn, const char *reason,
                                 NULL);
 }
 
+/** Get the full filename for storing the client auth credentials for the
+ *  service in <b>onion_address</b>. The base directory is <b>dir</b>. */
```

Comment:
Seems this can never fail that is never return NULL. I would add a comment about that.

Reply:
Fixed in e7f8319022.